eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/primitive/ByteHashSet.java

```diff
@@ -968,6 +971,38 @@ public void readExternal(ObjectInput in) throws IOException
         return result;
     }
 
+    @Override
+    public RichIterable<ByteIterable> chunk(int size)
+    {
+        if (size <= 0)
+        {
+            throw new IllegalArgumentException("Size for groups must be positive but was: " + size);
+        }
+        if (this.isEmpty())
```

Comment:
Same suggestion with reversing initialization and using notEmpty for if statement.